### DISCUSSION TITLE
Tailwind with CLOG?

### DISCUSSION THREAD

**Discussion by cdegroot** [rabbibotton/clog#282] [Q&A]
I'm wondering whether anyone has used Tailwind with CLOG. It smells (from some initial testing) that the builder is quite intertwined with W3 CSS, so I wonder whether it is possible at all and what the limitations would be. I never heard of W3 CSS before, but it smells like Tailwind would be a very good match (and I would venture a guess that it's a tad more mainstream, so it is usually simple to find an answer to "how do I ...?").

Context: at daytime, I use Elixir/LiveView with Tailwind, and I've gotten quite comfortable with developing decent UIs with it. I'm a developer, not a designer, so "mastering" one CSS utility library is pretty much as far as I'd like to go :-). I'm doing CL in the evening, and looking for options to add a UI to a project I'm working on; CLOG smells like just what I want in terms of close to what I'm comfortable with, design-wise.

**Comment by rabbibotton** [comment]
Tailwind seems to be close to same as w3 css, you should be able to load it in and use, although builder may look a little different. Just place on a panel html to load talwind's css.

**Comment by cdegroot** [comment]
There's two things here, I think - the builder style drop-downs seem to be married to W3 (which, by the way, I'm not saying is a bad technical choice or something, it just doesn't seem to be used in the part of the world I inhabit), I wonder whether it's easy to open that up so I can configure it showing Tailwind styles or just disable alltogether? And Tailwind has one feature that is quite important: the (mostly generated) CSS is very rich, so during deployment there's tooling involved to scan the source code for all style tags used and emit (minified) CSS with only these; I'm still prodding at CLOG and Builder so it may be a very dumb question as to where that tooling would go.

**Comment by rabbibotton** [comment]
I will take a look at it in the coming weeks and get you a best practice or demo using it with builder. (sorry mostly done with flooding issues, now just getting everything in life in running order, but will be on all soon)

**Comment by cdegroot** [comment]
No hurry and please don't go off into the deep end on my behalf - I was mostly trying to figure out whether someone had done it and what their experiences were :-)
